lib/mix/tasks/run_espec.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Mix.Tasks.RunEspec do
```

Comment:
Didn't want name confusion because there is already existent `espec` task in test environment